src/frontend/components/pages/new-ip/index.tsx

```diff
@@ -340,9 +377,67 @@ const NewIPButton: React.FC<NewIPButtonProps> = ({ principal }) => {
   if (!queriedUser || queriedUser.length === 0) return null;
 
   const onList = () => {
-    navigate("/bips");
+    setIsListModalOpen(true);
   };
 
+  const { call: approveBip721Transfer } = bip721LedgerActor.useUpdateCall({
+    functionName: "icrc37_approve_tokens",
+  });
+  const { call: listIntProp } = backendActor.useUpdateCall({
+    functionName: "list_int_prop",
+  });
+
+  const triggerList = (intPropId: bigint, sellPrice: bigint) => {
```

Comment:
I think you should be able to use https://github.com/bipquantum/bip721x/blob/feature/new-ui-deployment/src/frontend/components/hooks/useListIntProp.tsx instead, it's going to be easier